### ISSUE TITLE
Allow adding a dat.json file inside a directory

### ISSUE THREAD

**Issue by mitar** [soyuka/dat-daemon#8]
And from that gather all information about the dat URL and which directory corresponds to it.

**Comment by soyuka** [comment]
I don't think I get that correctly, what would be the steps?

1. read a directory
2. if there is a dat.json, parse it and find the dat to add?

IMO it's out of scope. You may have another tool that just parses a `dat.json` and instruct the daemon to add stuff via tcp?

**Comment by mitar** [comment]
No, the steps would be `datdaemon add /path/to/dat.json`. And then `datdaemon` would read `dat.json`, get URL from `url` field, and use `/path/to/` for directory.

Currently I am using [this script](https://github.com/soyuka/dat-daemon/issues/9#issuecomment-367392540) but I I really think reading a standard dotproject manifest file should be part of this project here.

Then I could do: `find . -name dat.json -exec datdaemon add {} ';'`

**Comment by mitar** [comment]
What is resolution here?
